src/components/SectionCard.js

```diff
@@ -1,16 +1,18 @@
 import CircleGauge from './CircleGauge';
 
-function SectionCard({ className, header, text, gaugeSize, percentage, color }) {
+function SectionCard({ className, header, text, gaugeSize, percentage, color, colspan, onClick }) {
```

Comment:
Is there any reason we can't collapse `colSpan` and className by replacing the div container with the button container altogether? If we can't, we should rename colSpan to more directly indicate its use as a className.

Reply:
Yeah, so for some reason when I just converted the section card div to a button, it screwed with the styling a bunch, so I went with the div inside a button approach and that retained the previous styling, but the colspan needs to be on the outermost tag to properly fit in the main viz layout. If you know of of a better way to do that let me know